SU-HSDO/suhumsci
# HSD8-1682:  Add additional Tugboat previews and update Tugboat passwords

# READY FOR REVIEW

## Summary

- Added history and west as additional tugboat previews with no default user createde
- Updated the default user password for traditional and colorful tugboat preview sites
- Added some additional visual diff urls


## Need Review By (Date)
10/23

## Urgency
low

## Steps to Test
1. Visit these three URLs and login with username: `tugboat` and password: `pushcar`

- https://pr1650-vzvgfo3kbusbf26eiopbmge2ixmll1i6.tugboatqa.com/user
- https://hs-colorful-vzvgfo3kbusbf26eiopbmge2ixmll1i6.tugboatqa.com/user
- https://hs-traditional-vzvgfo3kbusbf26eiopbmge2ixmll1i6.tugboatqa.com/user

2. Visit these URLs and confirm that they take you to replications of their live sites. 

- https://history-vzvgfo3kbusbf26eiopbmge2ixmll1i6.tugboatqa.com/
- https://west-vzvgfo3kbusbf26eiopbmge2ixmll1i6.tugboatqa.com/

3. Visit these URLs and confirm that you can not login with username: `tugboat` and password: `pushcar`

- https://history-vzvgfo3kbusbf26eiopbmge2ixmll1i6.tugboatqa.com/user
- https://west-vzvgfo3kbusbf26eiopbmge2ixmll1i6.tugboatqa.com/user

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
